src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -59,10 +59,14 @@ private static ArgumentParser getArgumentParser() {
                 .metavar("LOCATION")
                 .help("The GitHub URL or disk locations to clone repository.");
 
-        mutexParser.addArgument("-view")
+        parser.addArgument("-view")
+                .nargs("?")
                 .metavar("PATH")
                 .type(new ReportFolderArgumentType())
-                .help("Starts a server to display the dashboard in the provided directory.");
+                .setConst(Paths.get(""))
```

Comment:
`Paths.get("")` can be refactored to an expressive variable make `Line 122` more readable. :-P